core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java

```diff
@@ -48,7 +48,7 @@ public class ConsensusCommitConfig {
   private final boolean isIncludeMetadataEnabled;
 
   public ConsensusCommitConfig(DatabaseConfig databaseConfig) {
-    if (databaseConfig.getProperties().containsValue("scalar.db.isolation_level")) {
+    if (databaseConfig.getProperties().containsKey("scalar.db.isolation_level")) {
```

Comment:
Found a bug here. Fixed it as well. Also fixed the same bug in other places.

Reply:
Should it be a bugfix PR and cherry-picked to bugfix releases? (to 3.4.x and 3.5.x)

Reply:
I didn't think so actually because it's a trivial bug where the warning message isn't shown even when the old config is used. Do you think we should backport it?

Reply:
Ah, if it's just a warning message thing, then it can be fine as it is.